src/main/java/com/labsynch/labseer/service/StandardizationService.java

```diff
@@ -29,6 +29,8 @@ String getStandardizationDryRunReportFiles(StandardizationDryRunSearchDTO search
 	String executeStandardization(String username, String reason)
 			throws IOException, CmpdRegMolFormatException, StandardizerException;
 
+	public int restandardizeLots(List<Long> parentIds);
```

Comment:
How strange that this function existed but wasn't exposed. Just a nit - the term "restandardize" now has a specific connotation of running chemical structures through a "standardizer" or "preprocessor" or "structure cleaner", which isn't what this function does. It'd be nice to name this something more obvious, like "recalculateLotMolWeights"